AccountService.js

```diff
@@ -3,7 +3,7 @@
  */
 'use strict';
 
-import axios from 'axios';
+import {httpClient} from '@digitalbazaar/http-client';
 
 const headers = {Accept: 'application/ld+json, application/json'};
```

Comment:
these are the same headers already baked into http-client.  So if this is just being passed in everywhere, it can be safely removed along with the headers parameter to http-client.

https://github.com/digitalbazaar/http-client/blob/master/main.js#L6-L8

Reply:
Removed.